skafka/src/main/scala/com/evolutiongaming/skafka/package.scala

```diff
@@ -33,4 +36,17 @@ package object skafka {
   object Bytes {
     val Empty: Bytes = Array.empty
   }
+
+  private[skafka] implicit class NaturalTransformation[F[_], A](fa: F[A]) {
+    def liftTo[G[_]](implicit transform: F ~> G): G[A] =
+      transform(fa)
+  }
+
+  private[skafka] implicit def selfArrow[F[_]]: F ~> F = new (F ~> F) {
```

Comment:
could you explain this?

Reply:
This is obsolete weird stuff, thanks for the catch, deleted that.